pkgs/racket-doc/syntax/scribblings/parse/experimental.scrbl

```diff
@@ -346,4 +346,42 @@ the context above; instead, @racket[let-values] would report an
 invalid binding list.
 }
 
+@deftogether[
+ ((defthing prop:template-metafunction struct-type-property?)
+  (defthing template-metafunction? (-> any/c boolean?)))]{
+
+ A structure type property, and the associated predicate. The property value is
+ one of:
+
+ @itemlist[
+   @item{an identifier bound to a run-time metafunction}
```

Comment:
I think the phrase "run-time metafunction" is a little confusing. Maybe "a run-time procedure implementing the metafunction", and then below "The metafunction's implementation should accept ...".

It might be worth adding a paragraph somewhere about that clarifies that the "run time" of the metafunction is typically the "compile time" of the enclosing module.

Reply:
At first, I thought your comment was describing a different behavior—I interpreted the documentation as requiring the identifier to actually be bound to a metafunction (i.e. something ultimately bound by `define-template-metafunction`), not just any ordinary procedure. Upon closer inspection of the PR, though, you’re right: that’s not what the implementation does.

So, obviously, I agree with you: the docs as-written aren’t right. But it’s also worth thinking about which implementation is right. As implemented right now, without exporting the accessor (as both you and I have suggested), this means you can’t use `prop:template-metafunction` to easily indirect to an existing metafunction (assuming you don’t already have the metafunction procedure). Though, on the other hand, you could do it indirectly by making the metafunction procedure produce ``#`(other-metafunction-id . #,(stx-cdr input-stx)))``, so maybe it doesn’t matter.

Reply:
> I think the phrase "run-time metafunction" is a little confusing. Maybe "a run-time procedure implementing the metafunction", and then below "The metafunction's implementation should accept ...".

Done :)

I'm marking this as resolved based on the assumption that the difference between passing directly a procedure and passing an identifier previously defined using define-template-metafunction isn't too significant (and IIRC it's more common in this sort of prop:xyz to pass a procedure?). But @lexi-lambda please to reopen this conversation if you'd like the other implementation :)